lib/eventum/class.notification.php

```diff
@@ -1019,7 +1019,7 @@ public function notifySubscribers($issue_id, $emails, $type, $data, $subject, $i
                     $full_subject = "[#$issue_id] $subject: $extra_subject";
                 }
             } elseif (($type == 'new_issue') && ($is_assigned)) {
-                $full_subject = "[#$issue_id] New Issue Assigned: " . $data['iss_summary'];
+                $full_subject = "[#$issue_id] " . ev_gettext('New Issue Assigned') . ": " . $data['iss_summary'];
```

Comment:
translation must be whole sentence not individual keywords. also use sprintf positions so that different translations are allowed to reorder. also add hint for translators.

so:

```
// TRANSLATORS: %1 - issue_id, %2: issue summary
$full_subject = ev_gettext('[#%1$s] New Issue Assigned: %2$s' ,$issue_id, $data['iss_summary']);
```